image.md

```diff
@@ -0,0 +1,309 @@
+  * `layers`: This OPTIONAL property contains references to the layers that make up the root filesystem of the image.
+  This property MUST be a JSON encoded array where each entry consists of a REQUIRED `name` and OPTIONAL `ID` properties.
+  The array MUST be in the order in which the images are to be "layed down" into the filesystem bundle that is generated from this image.
+  In other words, the first image in the array is written to the filesystem first, and subsequent images are overlayed on top - potentially overlaying an earlier image's files.
+  If a referenced layer is embedded within this image then an `ID` property MUST be included.
+  If a referenced layer is not embedded within this image then an `ID` property is OPTIONAL.
+  * `root/path`: ThiS REQUIRED property, defined within [`config.json`](config.md), is the name of the directory into which all layers MUST appear.
+  While the `manifest.json` file extends the `config.json` file, a compliant OCI image is NOT REQUIRED to have all of the required fields from `config.json` within its `manifest.json` file if that image is only used as a layer (dependent image) within other images.
+  This is due to the fact that a layer's `manifest.json` properties are not used to produce a container's `config.json` file and therefore not all of them are used.
+  However, if an image is used to create a container then all of the "required" fields of the `config.json` MUST be present in the `manifest.json`.
+
+1. Layers.
+Each embedded layer MUST have a corresponding image (tar) file, at the root of the image.
+Each layer's tar file MUST be named with its ID followed by the appropriate file-extension based on how the tar file is written (e.g. `.tar` for uncompressed).
+
+1. `rootfs` directory.
+An image MAY optionally contain a directory that's the same as the `root/path` property of the `manifest.json` file.
+And if so, then it MUST be treated as the top-most layer of the root filesystem.
+
+1. There MAY be additional files within the image.
+This specification places no requirements on these files.
+A compliant OCI implementation is NOT REQUIRED to materialize these files into a container's filesystem bundle directory.
+
+## Serialization of an Image
+
+An image MUST be a `tar` of the files that make up an image as specified in the [Image Contents](#image-contents) section.
+The name of the tar file MUST be the ID of the image followed by the appropriate extension based on the type of tar file (e.g. `.tar` for uncompressed tar).
+The ID of an image MUST be a cryptographic hash of the uncompressed tar file.
+An `ID` MUST be a string of the form `<algorithm>-<hash value>`.
+The `algorithm` part of the string MUST be in lower-case.
+For example, `sha512` not `SHA512`.
+It is RECOMMENDED that implementations use the sha512 algorithm.
```

Comment:
We should perhaps back this with some reasoning and references and also suggest alternatives.